custom/icds_reports/tests/agg_tests/reports/test_prevalence_of_stunting.py

```diff
@@ -52,69 +52,27 @@ def test_map_data(self):
         self.assertDictEqual(
             data['data'],
             {
-                'st4': {
```

Comment:
Some state data is to deleted from the test after this change. Which is valid I think. But would love if you could dig and try to find out why they actually got removed after this change(as a secondary/low priority task in your list). This would help you realize what this change was actually doing at database level. @sudh-kvantum

Reply:
Alright. Thanks